generator/template_repo.go

```diff
@@ -153,7 +155,9 @@ func (t *Repository) LoadDir(templatePath string) error {
 		if strings.HasSuffix(path, ".gotmpl") {
 			assetName := strings.TrimPrefix(path, templatePath)
 			if data, err := ioutil.ReadFile(path); err == nil {
-				t.AddFile(assetName, string(data))
+				if err := t.AddFile(assetName, string(data)); err != nil {
```

Comment:
instead of panicing can you return an error? or is the stack trace really relevant?